app/views/layouts/engine.html.erb

```diff
@@ -12,11 +12,7 @@
 <% end %>
 
 <%= render layout: 'layouts/unconstrained' do %>
-  <div class="l-constrained">
```

Comment:
For example https://www.moneyadviceservice.org.uk/en/tools/mortgage-calculator relies on `l-container-tool` being there